Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+ARG JAVA_VERSION=17
+ARG TARGETPLATFORM
```

Comment:
it's not used here but I would use it by adding the section about installing `tini` as it's done for the Strimzi HTTP bridge.